tests/utils/constants.py

```diff
@@ -3,14 +3,15 @@
 
 import pytest
 
-from kakarot_scripts.constants import BLOCK_GAS_LIMIT, MAX_SAFE_CHAIN_ID
+from kakarot_scripts.constants import BLOCK_GAS_LIMIT
 
 BLOCK_GAS_LIMIT = BLOCK_GAS_LIMIT
 MIN_BASE_FEE_PER_BLOB_GAS = 1
 
 CHAIN_ID = int.from_bytes(b"KKRT", "big")  # KKRT (0x4b4b5254) in ASCII
 BIG_CHAIN_ID = int.from_bytes(b"SN_SEPOLIA", "big")
-MAX_SAFE_CHAIN_ID = MAX_SAFE_CHAIN_ID
+# see https://gist.github.com/rekmarks/a47bd5f2525936c4b8eee31a16345553
+MAX_SAFE_CHAIN_ID = 4503599627370476
```

Comment:
We cannot use it everywhere because: 
* for our deployment on mainnet / sepolia we want to support ledger
* for hive tests we want to support very big chain id